src/components/DNSenumTool/DNSenumTool.tsx

```diff
@@ -32,6 +32,8 @@ const DnsenumTool = () => {
     const [pid, setPid] = useState("");
     const [allowSave, setAllowSave] = useState(false);
     const [hasSaved, setHasSaved] = useState(false);
+    const [missingCommands, setMissingCommands] = useState<string[]>([]);
```

Comment:
The checkAllCommandsAvailability and installationModal components require some specific variables. 
```suggestion
    const [isCommandAvailable, setIsCommandAvailable] = useState(false); // State variable to check if the command is available.
    const [opened, setOpened] = useState(!isCommandAvailable); // State variable that indicates if the modal is opened.
    const [loadingModal, setLoadingModal] = useState(true); // State variable to indicate loading state of the modal.
    
    
```